hyperspy/_components/expression.py

```diff
@@ -18,7 +18,6 @@
 
 from functools import wraps
 import numpy as np
-import sympy
```

Comment:
wrong import removed, fixed in https://github.com/hyperspy/hyperspy/pull/2977/commits/c6861fbf54c4d1457f8085a40773ab41a7ab383f